upload/upload.go

```diff
@@ -132,7 +134,7 @@ func checkTokenExpiration(accessToken string) (bool, error) {
 }
 
 // Function uploadFiles uploads the files in the input list to the s3 bucket
-func uploadFiles(files, outFiles []string, config *Config) error {
+func uploadFiles(files, outFiles []string, outDir string, config *Config) error {
```

Comment:
Shouldn't `outDir` be `targetDir` also in the code?

Reply:
Yes, done.